api/app/models/task.rb

```diff
@@ -1,2 +1,11 @@
 class Task < ApplicationRecord
+  belongs_to :user
+
+  def self.send_duedate_reminders
+    Task.all.each do |task|
+      if Date.today == task.duedate.prev_day && task.complete == false
```

Comment:
Totally works, but this is more idiomatic in Ruby:
```
if Date.today == task.duedate.prev_day && !task.complete?
```

Reply:
Ahhhh yea I love that.